dev/vision-n-scope/7-contributions.md

```diff
@@ -16,7 +16,39 @@ placeholder
 
 ## Alex Marx
 
-placeholder
+### Introduction
+
+By the end of Capstone, I'd like to have worked on and completed the game server which deals with taking in requests for new games and then spawning virtual machines per the instructions. I would like to refer to the low level work I have done on the game server with creating hypervisors and other such tools on my resume.
+
+### System Architecture
```

Comment:
Remove the heading and subheadings about the system architecture, and instead turn the captions into their own paragraph in the Introduction setion